EventListener/EmailConfirmationListener.php

```diff
@@ -58,6 +61,10 @@ public static function getSubscribedEvents()
      */
     public function onRegistrationSuccess(FormEvent $event)
     {
+        if (!$this->confirmationEnabled) {
```

Comment:
is this needed ? The listener is not registered at all when the confirmation is not enabled AFAIK